### PR TITLE
Add an example of how to use drand's randomness

### PR DESCRIPTION
**Pull request by benr-ml**

### CODE REVIEW

File: sui_programmability/examples/games/sources/drand_random_selection.move [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+            round,
+            status: IN_PROGRESS,
+            participants: 0,
+            winner: option::none(),
+        };
+        transfer::share_object(game);
+    }
+
+    /// Anyone can close the game by providing the randomness of round-2.
+    public entry fun close(game: &mut Game, drand_sig: vector<u8>, drand_prev_sig: vector<u8>) {
```
Comment (kchalkias) [inline sui_programmability/examples/games/sources/drand_random_selection.move:103]: How about using the unchained version and document it properly? I think it's easier and cheaper (in gas and tx size) to demonstrate at this stage.

Comment (benr-ml) [reply sui_programmability/examples/games/sources/drand_random_selection.move:103]: The public api currently only serve the chained mode

Comment (kchalkias) [reply sui_programmability/examples/games/sources/drand_random_selection.move:103]: Pfff.. it's a pity but ok fair enough.

File: sui_programmability/examples/games/sources/drand_random_selection.move [hunk 2]
```diff
@@ -0,0 +1,215 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// A basic game that depends on randomness from drand.
```
Comment (kchalkias) [inline sui_programmability/examples/games/sources/drand_random_selection.move:4]: Let's mention clearly that we use the chained version.

**Review by kchalkias — APPROVED** [review]

**Pull request merged**